content/en/docs/setup/other_config/features/notifications/index.md

```diff
@@ -123,6 +124,23 @@ hal config notification twilio enable
 echo $TWILIO_AUTH_TOKEN | hal config notification twilio edit --account $TWILIO_ACCOUNT_SID --from $TWILIO_PHONE_NUMBER --token
 ```
 
+
+## CDEvents
+
+To enable CDEvents notification support, add the following statement to the `echo-local.yml` file:
+
+```yaml
+cdevents:
+  enabled: true
+```
+
+Spinnaker supports sending CDEvents to [Message Broker](https://www.enterpriseintegrationpatterns.com/patterns/messaging/MessageBroker.html)
```

Comment:
"to a" vs. "to"

Reply:
Are there any docs on configuring the target destination though as part of this?

Reply:
Sorry rephrase:  Can you maybe mention "The target broker is defined in the notification trigger itself".  I WOULD be... cautious given that this is an arbitrary publish of payload data to a URL, similar to how webhooks operate.

Reply:
Thank you @jasonmcintosh for reviewing this PR.
May be I can share the configuration for [Knative Eventing](https://knative.dev/docs/getting-started/) as a Message Broker